### PR TITLE
Adding localhost proxy to StorybookJS so we can make API calls

### PR DESCRIPTION
**Pull request by iisa**
Problem:
Making AJAX calls inside of a StorybookJS component will cause a CORS error

Solution:
Piggyback on StorybookJS' dev server and route external calls through a proxy

### CODE REVIEW

File: packages/ia-components/.storybook/middleware.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const proxy = require('http-proxy-middleware');
```
Comment (shaneriley) [inline packages/ia-components/.storybook/middleware.js]: I'd refactor this file to make it easier to both change the base URL for the API (in the event of a dev or staging env) and to quickly add routes, like:

```
const base_url = "https://archive.org";

const constructOptions = (route) => (
  {
    target: `${base_url}/${route}`,
    logLevel: 'debug',
    changeOrigin: true,
    pathRewrite: {
      `^/${route}`: `${base_url}/${route}`
    }
  }
);

module.exports = function expressMiddleware (router) {
  ["download", "images", "serve"].forEach((route) => {
    router.use(`/${route}`, proxy(constructOptions(route)))
  });

  router.use('/api', proxy({
    target: base_url,
    logLevel: 'debug',
    changeOrigin: true,
    pathRewrite: {
      '^/api': base_url
    }
  }));
};
```

File: packages/ia-components/package.json [hunk 2]
```diff
@@ -21,8 +21,10 @@
         "@storybook/addon-knobs": "^4.1.4",
         "@storybook/react": "^4.1.4",
         "assert": "^1.4.1",
+        "axios": "^0.18.0",
```
Comment (shaneriley) [inline packages/ia-components/package.json]: I don't see Axios being used in this commit.

File: packages/ia-components/.storybook/middleware.js (hunk 1 above)
Comment (mc2) [reply packages/ia-components/.storybook/middleware.js]: Great contribution, Shane. Agree.

Comment (iisa) [reply packages/ia-components/.storybook/middleware.js]: that's a good optimization, will apply it.

**Pull request merged**